singer_sdk/sinks/core.py

```diff
@@ -51,6 +49,8 @@ class Sink(metaclass=abc.ABCMeta):
     logger: Logger
 
     MAX_SIZE_DEFAULT = 10000
+    CHECK_RECORD_FORMATS = False
```

Comment:
@edgarrmondragon I thinking this need to not be a constant since developers will be changing it as needed.  I was thinking of changing it to `allow_record_format_cheks`.  Should it stay here or move?  Please let me know what you think.

Reply:
Agreed. How about `validate_field_string_format`?